packages/rpc-graphql/src/schema/type-defs/account.ts

```diff
@@ -215,6 +215,13 @@ export const accountTypeDefs = /* GraphQL */ `
         lockCpi: Boolean
     }
 
+    """
+    Token-2022 Extension: Unparseable
+    """
+    type SplTokenExtensionUnparseable implements SplTokenExtension {
+        extension: String
+    }
```

Comment:
@buffalojoec, I snuck this in here, because the current test fixtures actually produce an unparseable extension as of Agave 2.2.